Use crm_attribute --list-options (#1379)
ClusterLabs/crmsh: doc/crm.8.adoc

==== `info` (`meta`)

Show the meta-data of a resource agent type. This is where users

> can find information on how to use a resource agent. To get cluster
> option meta-data, use the `cluster` type. To get pacemaker-fenced
> meta-data, use the `pacemaker-fenced` type.

Reviewer: ```suggestion
can find information on how to use a resource agent. It is also
possible to use the `pacemaker-fenced` type to get the corresponding
meta-data information.
```

info apache
info ocf:pacemaker:Dummy
info stonith:ipmilan

> info cluster

Reviewer: ```suggestion
```

If you agree, we could delete and replace "crm ra info cluster" to "crm configure property info".

Author: I prefer not to hide it from help info, just don't show it from completion result
